yarn.lock

```diff
@@ -41,11 +45,16 @@
     webpack-filter-warnings-plugin "1.2.1"
     webpack-rtl-plugin "1.8.0"
 
-"@automattic/calypso-color-schemes@1.0.0", "@automattic/calypso-color-schemes@^1.0.0":
+"@automattic/calypso-color-schemes@1.0.0":
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/@automattic/calypso-color-schemes/-/calypso-color-schemes-1.0.0.tgz#17a14e3257bd90b40e960d624cca4353d4d20c34"
   integrity sha512-W7r4pgcBauLx65oTLbHKV84ScnfEKmW7T/sXkPfByHhuIE7+OpubmiiBsizWatk1I/puUTmj+SDqEOJMOyRdzA==
 
+"@automattic/calypso-color-schemes@^2.0.2":
```

Comment:
@ockham This may be related to the visual change you're seeing…
https://github.com/Automattic/jetpack/pull/13503#pullrequestreview-292105012

Reply:
Yeah… I think this line:

https://github.com/Automattic/jetpack/blob/f55f55e67b1071518ece4318a349e359acf88309/extensions/shared/components/upgrade-nudge/style.scss#L6

Would need to become [`--studio-white`](https://github.com/Automattic/color-studio/blob/4b3262ac652a39c434c818e9e9c357fc23c8c0cc/dist/color-properties.css#L4).

Reply:
@porada :wave: where there any other colors than `white` that now need `--studio` prefix?

Bummer with CSS variables (as opposite to scss variables) is that they fail silently. :-/

Checked if these would be listed on changelog but no dice: 
https://github.com/Automattic/color-studio/blob/4b3262ac652a39c434c818e9e9c357fc23c8c0cc/CHANGELOG.md

Reply:
We replaced white with 3 semantic properties:

* `--color-surface` (default white background),
* `--color-text-inverted` (white text, which is also the correct substitute for the case mentioned above),
* `--color-border-inverted` (white border).

Generally, we should avoid `--studio-*` values in favor of the [theme properties](https://github.com/Automattic/wp-calypso/blob/master/packages/calypso-color-schemes/src/shared/color-schemes/_default.scss) (that’s a full listing).

---

> Bummer with CSS variables (as opposite to scss variables) is that they fail silently

Yes, that is a disadvantage of using CSS properties. I hope we can make Stylelint throw errors when using an undefined property.

> Checked if these would be listed on changelog but no dice

Color Studio’s changelog focuses only on color value changes. [`calypso-color-schemes`](https://github.com/Automattic/wp-calypso/blob/master/packages/calypso-color-schemes/CHANGELOG.md) would be a better place to look this up.

Reply:
Gotcha, thanks for a thorough overview @porada !

Reply:
Opened an issue to separate colors from `calypso-build`: https://github.com/Automattic/wp-calypso/issues/36466